lib/blob_store/src/bitmask/mod.rs

```diff
@@ -217,7 +217,7 @@ impl Bitmask {
     }
 
     /// The amount of blocks that are available for reuse in the page.
-    #[allow(dead_code)]
+    #[allow(dead_code)] // required
```

Comment:
I feel like the required note is obsolete, since the allow attribute should not be there if not required.

Assuming you use them now to keep track of what you've checked, do you remove them in this PR when having covered all?

Reply:
PR is still in draft. 😂

I've added them temporarily (so they all show up in git diff, and it's easier to navigate), and plan to remove them later. I just wanted to make a CI run, to see if any of the annotations I've removed will be triggered on CI somewhere.

Reply:
> Assuming you use them now to keep track of what you've checked, do you remove them in this PR when having covered all?

Yeah, exactly this. 😂